CMake/Dependencies/libkvspic-CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ ExternalProject_Add(libkvspic-download
 	BINARY_DIR        "${CMAKE_CURRENT_BINARY_DIR}/kvspic-build"
 	CMAKE_ARGS
 	  -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+	  -DKVS_DEFAULT_STACK_SIZE=${KVS_DEFAULT_STACK_SIZE}
```

Comment:
Where is `KVS_DEFAULT_STACK_SIZE` defined?

Reply:
By other CMake arguments from other repos. See webrtc-c as an example